src/core/raps/actions/crosschainSwap.test.ts

```diff
@@ -5,11 +5,10 @@ import {
   QuoteError,
   getCrosschainQuote,
 } from '@rainbow-me/swaps';
-import { mainnet } from 'viem/chains';
 import { beforeAll, expect, test } from 'vitest';
 
 import { connectedToHardhatStore } from '~/core/state/currentSettings/connectedToHardhat';
-import { ChainId } from '~/core/types/chains';
+import { ChainId, chainHardhat } from '~/core/types/chains';
```

Comment:
see above comment. had to make these changes to get them to pass with new mocking setup.